src/lib/Server/Output/ValueObjectVisitor/Location.php

```diff
@@ -23,9 +26,15 @@ class Location extends ValueObjectVisitor
      */
     private $locationService;
 
-    public function __construct(LocationService $locationService)
+    /**
+     * @var \eZ\Publish\API\Repository\ContentService
+     */
+    private $contentService;
+
+    public function __construct(LocationService $locationService, ContentService $contentService)
```

Comment:
Doesnt the service definition have to be updated as well?

Reply:
Yup, not sure how was it skipped.